controller/api/public/grpc_server.go

```diff
@@ -31,8 +31,13 @@ type (
 	}
 )
 
+type podReport struct {
+	lastReport              time.Time
+	processStartTimeSeconds time.Time
+}
+
 const (
-	podQuery                   = "count(process_start_time_seconds) by (pod)"
+	podQuery                   = "sum(process_start_time_seconds{%s}) by (pod, namespace)"
```

Comment:
related to my earlier comment, `podQuery` is used in `SelfCheck`: https://github.com/runconduit/conduit/blob/13aaa82c9502458b8f4a34ddb180a76f8f203d81/controller/api/public/grpc_server.go#L69-L70

Reply:
ahhhh good find! thanks!